tools/eslint-rules/no-unescaped-regexp-dot.js

```diff
@@ -4,48 +4,20 @@
  */
 'use strict';
 
-const path = require('path');
-const utilsPath = path.join(__dirname, '..', 'eslint', 'lib', 'ast-utils.js');
-const astUtils = require(utilsPath);
-const getLocationFromRangeIndex = astUtils.getLocationFromRangeIndex;
-const getRangeIndexFromLocation = astUtils.getRangeIndexFromLocation;
-
 //------------------------------------------------------------------------------
 // Rule Definition
 //------------------------------------------------------------------------------
 
 module.exports = function(context) {
-  const sourceCode = context.getSourceCode();
+  const s = context.getSourceCode();
```

Comment:
Is it sufficiently more desirable to have a descriptive variable name that we're OK that it introduces a line break into the object literal (to wrap at 80 chars due to the descriptive variable name) on line 20?

I'm OK either way. Just want to make sure the cost of the descriptive variable name is taken into account.